src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/BufferWriterStream.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Buffers;
+using System.IO;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.SignalR.Internal.Protocol
+{
+    public sealed class BufferWriterStream : Stream
```

Comment:
We don't need this stream, we already have the PipeWriter one.

Reply:
Could you show me what you mean? PipeWriter only implements `IBufferWriter<byte>`. And the file you linked me to yesterday was internal.

Reply:
It's a shared file that's currently imported on in Sockets.Http and Sockets.Http.Client. See https://github.com/aspnet/SignalR/blob/bb59bb0a12f0872f36cbd050ec993ef5be9d29ae/src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs#L455 (this could be optimized btw we allocate per send request right now. You only need one per connection).